src/Model/PowerElectronics/InductionMotor/InductionMotor.cpp

```diff
@@ -26,7 +26,7 @@ InductionMotor<ScalarT, IdxT>::InductionMotor(IdxT id, ScalarT Lls, ScalarT Rs,
     Llr_(Llr),
     Rr_(Rr),
     Lms_(Lms),
-    RJ_(RJ),
+    Rjac_(RJ),
     P_(P)
```

Comment:
I suggest explaining what `Rjac_` is:
```suggestion
    Rjac_(RJ), // initialize mysterious Rjac_ member
    P_(P)
```

Reply:
I am not sure myself, I didn't write this function. It is the result of a search and replace to make variable names consistent.

Reply:
This variable is not `RJac` but is in fact `RJ`. This comes from a InductionMotor equations provided to me by @pelesh so I am not sure on its physical meaning.

Reply:
The way it is used in the equations, it looks as it is rotor moment of inertia. In the documentation, it is denoted with "J".

Reply:
The "rotor" explains where the "R" if from. But I agree that the name lacks clarity. J would be no better. What do you suggest? Perhaps `rot_mom_intert` or something? Come to think of it, none of these follow naming conventions. Is there a good reason for that?